src/datasource.ts

```diff
@@ -122,10 +122,9 @@ export class OpenSearchDatasource extends DataSourceApi<OpenSearchQuery, OpenSea
         if (err.data) {
           const message = err.data.error?.reason ?? err.data.message ?? 'Unknown error';
 
-          throw {
-            message: 'OpenSearch error: ' + message,
-            error: err.data.error,
-          };
+          let newErr = new Error('OpenSearch error: ' + message);
```

Comment:
I needed to change this to an `Error` type to catch it properly in `OpenSearchDetails.tsx`